### PR TITLE
Include March 2024 Annual Update blog post

### PR DESCRIPTION
**Pull request by trvrb**
Include March 2024 Annual Update Blog Post

### CODE REVIEW

**Review by jameshadfield — APPROVED** [review]

File: static-site/content/blog/2024-03-27-annual-update-march-2024.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Development on **nextstrain.org** focused on sharing and surfacing datasets. We have supported ["community" datasets](https://docs.nextstrain.org/en/latest/guides/share/community-builds.html) since 2018 through the ability to display phylogenetic analyses deposited on GitHub. More recently, the Seattle team has implemented an integrated solution called ["Nextstrain Groups" where users can maintain their own public or private analyses](https://docs.nextstrain.org/en/latest/learn/groups/index.html). There are currently 41 user groups, including public health authorities, with groups datasets managed via the CLI. The same credentials to login via `nextstrain login` are used to login to nextstrain.org via browser. We've recently made improvements to AWS infrastructure and permissions to streamline inclusion of new groups, but this still requires individuals to reach out to us via email. We've provided a whitelisted version of Nextstrain Groups for the US CDC to use internally. In addition, we've implemented new functionality to surface ["snapshots" of historical datasets](https://docs.nextstrain.org/en/latest/guides/snapshots.html) on nextstrain.org.
+
```
Comment (victorlin) [inline static-site/content/blog/2024-03-27-annual-update-march-2024.md:34]:
```diff
- We've provided a whitelisted version of Nextstrain Groups for the US CDC to use internally.
+ We've provided a white-labeled version of Nextstrain Groups for the US CDC to use internally.
```

File: static-site/content/blog/2024-03-27-annual-update-march-2024.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Development of **Auspice tree visualization** has focused on inclusion of a ["measurements panel"](https://www.frontiersin.org/articles/10.3389/fbinf.2023.1069487/full) intended for displaying titer or DMS data and on improvements to the "entropy panel" to support more complex genome architectures.
+
+In terms of **training and technical support** we've provided standalone installers for Linux, Mac, and Windows (allowing one-command installation) and new managed runtimes for Conda and Singularity, centralized documentation and tutorials at [docs.nextstrain.org](https://docs.nextstrain.org), hosted weekly drop in office hours via Zoom, responded to user emails to hello@nextstrain.org and responded to posts to discussion board [discussion.nextstrain.org](https://discussion.nextstrain.org). We've also led hands-on training sessions at VEME in Aug 2022 and Aug 2023 and hosted intern Alhaji Olono from ACEGID in summer 2023 to work on Nextstrain build for African Swine Fever Virus.
+
```
Comment (victorlin) [inline static-site/content/blog/2024-03-27-annual-update-march-2024.md:38]:
```diff
- to work on Nextstrain build for African Swine Fever Virus
+ to work on a Nextstrain build for African Swine Fever Virus
```

File: static-site/content/blog/2024-03-27-annual-update-march-2024.md (hunk 1 above)
Comment (victorlin) [inline static-site/content/blog/2024-03-27-annual-update-march-2024.md:34]: > There are currently 41 user groups

Where does this number come from? I count 62 groups (33 public, 29 private) in [env/production/groups.json](https://github.com/nextstrain/nextstrain.org/blob/e79e294d34ac0e816e870f8dabb39fa8f1aee8f2/env/production/groups.json).

**Review by victorlin — CHANGES_REQUESTED** [review]
Suggesting a few small tweaks and clarifying a metric.

File: static-site/content/blog/2024-03-27-annual-update-march-2024.md (hunk 1 above)
Comment (trvrb) [reply static-site/content/blog/2024-03-27-annual-update-march-2024.md]: Resolved in 47b6ab322fca3125cdee07a6b1dcc7fc07c47b0a

File: static-site/content/blog/2024-03-27-annual-update-march-2024.md (hunk 2 above)
Comment (trvrb) [reply static-site/content/blog/2024-03-27-annual-update-march-2024.md]: Resolved in 47b6ab322fca3125cdee07a6b1dcc7fc07c47b0a

File: static-site/content/blog/2024-03-27-annual-update-march-2024.md (hunk 1 above)
Comment (trvrb) [reply static-site/content/blog/2024-03-27-annual-update-march-2024.md]: Thanks for the catch! Resolved in 63119b669dced9db961fb99a87df3204ad01640f.

**Review by victorlin — APPROVED** [review]

**Pull request merged**